packages/flutterfire_ui/lib/src/i10n/lang/es.dart

```diff
@@ -0,0 +1,264 @@
+    this.smsAutoresolutionFailedError =
+        'No se pudo resolver el código SMS automáticamente. Por favor ingrese su código manualmente',
+    this.smsCodeSentText = 'Código SMS enviado',
+    this.sendingSMSCodeText = 'Enviando código SMS ...',
+    this.verifyingSMSCodeText = 'Verificando el código SMS ...',
+    this.enterSMSCodeText = 'Ingrese el código SMS',
+    this.emailIsRequiredErrorText = 'El correo electrónico es requirido',
+    this.isNotAValidEmailErrorText = 'Ingresa un Correo Electrónico válido',
+    this.userNotFoundErrorText = 'La cuenta NO existe',
+    this.emailTakenErrorText = 'La cuenta con ese correo electrónico ya existe',
+    this.accessDisabledErrorText =
+        'El acceso a esta cuenta se ha inhabilitado temporalmente',
+    this.wrongOrNoPasswordErrorText =
+        'La contraseña no es válida o el usuario no tiene contraseña.',
+    this.signInText = 'Ingresar',
+    this.registerText = 'Registrarse',
+    this.registerHintText = '¿No tienes una cuenta?',
+    this.signInHintText = '¿Ya tienes una cuenta?',
+    this.signOutButtonText = 'Salir',
+    this.phoneInputLabel = 'Número de teléfono',
+    this.phoneNumberInvalidErrorText = 'El número de teléfono no es válido',
+    this.phoneNumberIsRequiredErrorText = 'El número de teléfono es requerido',
+    this.profile = 'Perfil',
+    this.name = 'Nombre',
+    this.deleteAccount = 'Eliminar cuenta',
+    this.passwordIsRequiredErrorText = 'La contraseña es requerida',
+    this.confirmPasswordIsRequiredErrorText = 'Confirma tu contraseña',
+    this.confirmPasswordDoesNotMatchErrorText = 'Las contraseñas no coinciden',
+    this.confirmPasswordInputLabel = 'Confirma la contraseña',
+    this.forgotPasswordButtonLabel = '¿Olvidaste tu contraseña?',
+    this.forgotPasswordViewTitle = 'Olvidar contraseña',
+    this.resetPasswordButtonLabel = 'Restablecer contraseña',
+    this.verifyItsYouText = 'Verifica que eres tú',
+    this.differentMethodsSignInTitleText =
+        'Utilice uno de los siguientes métodos para iniciar sesión',
+    this.findProviderForEmailTitleText = 'Ingrese su correo electrónico para continuar',
+    this.continueText = 'Continuar',
+    this.countryCode = 'Código de país',
+    this.codeRequiredErrorText = 'El código de país es requerido',
+    this.invalidCountryCode = 'Código de país inválido',
+    this.chooseACountry = 'Seleccione un país',
+    this.enableMoreSignInMethods = 'Habilitar más métodos de inicio de sesión',
+    this.signInMethods = 'Métodos de inicio de sesión',
+    this.provideEmail = 'Proporcione su correo electrónico y contraseña',
+    this.goBackButtonLabel = 'Volver',
+    this.passwordResetEmailSentText =
+        'Le enviamos un correo electrónico con un enlace para restablecer su contraseña. Por favor revise su correo electrónico.',
+    this.forgotPasswordHintText =
+        'Proporcione su correo electrónico y le enviaremos un enlace para restablecer su contraseña',
```

Comment:
```suggestion
        'Introduce su correo electrónico y le enviaremos un enlace para restablecer su contraseña',
```

Reply:
I feel there is a mix of formal and informal language here. 

"Introduce" is informal, where "su" is formal.

It should be "introduce tu" (informal) or "introduzca su" (formal)

Reply:
Introduce is the same for 'tu' and 'usted'. Introduzca is the subjunctive form of the verb